crates/view/src/storage/schema.sql

```diff
@@ -54,7 +54,8 @@ CREATE TABLE tx (
     tx_hash                 BLOB PRIMARY KEY NOT NULL,
     tx_bytes                BLOB NOT NULL,
     block_height            BIGINT NOT NULL,
-    return_address          BLOB
+    return_address          BLOB,
+    memo_text               TEXT
```

Comment:
Schema change! This breaks existing clients. Testing locally against testnet, I do indeed see the error message:

> `Error: can't load view database created by client version 0.80.5 using client version 0.80.5: they have different schemata, so you need to reset your view database and resynchronize by running pcli view reset`

That's probably tolerable for a point-release, but we should be careful about downstream consequences. I'm thinking in particular of relayers like `hermes`, which will likely fail when their deps are upgraded to a more recent view crate. I see we already merged a schema change in https://github.com/penumbra-zone/penumbra/pull/4801 so we'll need to accommodate for the change already anyway. Relevant for rebasing the hermes fork, cc @zbuc.